src/Simplex/Chat/Messages.hs

```diff
@@ -764,7 +768,9 @@ instance FromJSON MsgDecryptError where
 
 ciRequiresAttention :: forall d. MsgDirectionI d => CIContent d -> Bool
 ciRequiresAttention content = case msgDirection @d of
-  SMDSnd -> True
+  SMDSnd -> case content of
+    CIInvalidJSON _ -> False
+    _ -> True
```

Comment:
Why make a differentiation here? It only affects chat timestamp in backend for snd items, and no chat items are going to be created in invalid json type, so it wouldn't even change that (and if they are created it's probably still correct to update chat ts)

```suggestion
  SMDSnd -> True
```